smslibrary/src/test/java/com/eis/smslibrary/TestSMSParseStrategy.java

```diff
@@ -0,0 +1,34 @@
+package com.eis.smslibrary;
+
+/**
+ * @author Riccardo De Zen
+ */
+class TestSMSParseStrategy implements SMSParseStrategy {
+
+    static final String KEY = "If violence is not the answer, the question itself is wrong.";
```

Comment:
Where did you get this from? Can't find any reference on the internet.

Reply:
I read it in a Berserk related Facebook group a long time ago, I guess it's a parody of other similar quotes because I never managed to find the source myself